### PR TITLE
Av 2 general sign up page

### PR DESCRIPTION
**Pull request by gunjansamrit**

### CODE REVIEW

File: avfrontend/src/App.js [hunk 1]
```diff
@@ -1,8 +1,51 @@
...
+  const [showSignUp, setShowSignUp] = useState(false);
+
+  const changeTab = (tabNumber) => {
+    setSelectedTab(tabNumber);
+  };
+
   return (
     <div className="App">
-      <h1>Arogya Varta Demo</h1>
+      <header>
```
Comment (Sudhanshu10Sathawane) [inline avfrontend/src/App.js]: create a different header component

File: avfrontend/src/App.js [hunk 2]
```diff
@@ -1,8 +1,51 @@
...
+        {/* Header content goes here */}
+        <h1>Arogya Vartha</h1>
+      </header>
+      <div className="main">
```
Comment (Sudhanshu10Sathawane) [inline avfrontend/src/App.js]: Create this component as signup component

File: avfrontend/src/App.js [hunk 3]
```diff
@@ -1,8 +1,51 @@
...
+          {/* Main content goes here */}
+          {selectedTab === 1 && <AdminLogin/>}
+          {selectedTab === 2 && <DoctorLogin/>}
+          {selectedTab === 3 && !showSignUp && <PatientLogin setShowSignUp={setShowSignUp} />}
+          {selectedTab === 3 && showSignUp && <PatientSignUp setShowSignUp={setShowSignUp} />}
+          {selectedTab === 4 && <RadiologistLogin/>} 
+          {selectedTab === 5 && <LabLogin/>} 
+        </div>
+      </div>
+      <footer>
```
Comment (Sudhanshu10Sathawane) [inline avfrontend/src/App.js]: For footer create component

File: avfrontend/src/logins/DoctorLogin.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    const { name, value } = e.target;
+    setFormData(prevState => ({
+      ...prevState,
+      [name]: value
+    }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(formData); // You can do further processing with the form data here
```
Comment (Sudhanshu10Sathawane) [inline avfrontend/src/logins/DoctorLogin.jsx]: comment or remove log

**Review by Sudhanshu10Sathawane — CHANGES_REQUESTED** [review]

File: avfrontend/src/App.js (hunk 3 above)
Comment (gunjansamrit) [reply avfrontend/src/App.js]: closse

**Review by Sudhanshu10Sathawane — APPROVED** [review]

**Review by stsiyer — APPROVED** [review]

**Pull request merged**